Gulpfile.js

```diff
@@ -2,7 +2,7 @@
 
 const gulp = require('gulp')
 const htmlhint = require('gulp-htmlhint')
-const sass = require('gulp-sass')
+const sass = require('gulp-sass')(require('sass'))
```

Comment:
This is consistent with the syntax on the `gulp-sass` page but I still don't like it, it looks too much like magic.

For better readability I would suggest:
```
const sassCompiler = require('sass')
const gulpSass = require('gulp-sass')
const sass = gulpSass(sassCompiler)
```
But that could just be more of a personal preference so I'll leave it up to you.

Reply:
Fair comment -- changed as suggested!